frontend/src/main/components/Nav/AppNavbar.js

```diff
@@ -23,6 +24,23 @@ export default function AppNavbar({currentUser, doLogout}) {
             }
           </Nav>
 
+          <Nav className="me-auto">
+            {
+              systemInfo && systemInfo?.springH2ConsoleEnabled && (
```

Comment:
```suggestion
              systemInfo?.springH2ConsoleEnabled && (
```
`systemInfo &&` is redundant here, the use of the optional chaining operator `?.` accomplishes this.